contrib/emacs/hurl-mode.el

```diff
@@ -0,0 +1,133 @@
+;;; hurl-mode.el --- Major mode for hurl  -*- lexical-binding: t; -*-
+
+;; Homepage: https://github.com/Orange-OpenSource/hurl
+;; Keywords: Hurl, shell
+
+;; Package-Version: 0.1.0
+;; Package-Requires: ((emacs "24"))
+
+;; SPDX-License-Identifier: Apache License 2.0
+
+;; This program is free software; you can redistribute it and/or modify
```

Comment:
Sorry @azzamsa everything looks good except this: can you remove line 11 to 22 (everything related to license). We should just set `SPDX-License-Identifier: Apache License 2.0` and nothing else as it is sufficient to make our license explicit. Thanks again!!

Reply:
Oh sorry. I missed this.